kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/SessionsResourceSuite.scala

```diff
@@ -171,4 +171,26 @@ class SessionsResourceSuite extends KyuubiFunSuite {
         assert(404 == response.getStatus)
     }
   }
+
+  test("test get infoType") {
+    val requestObj = SessionOpenRequest(
+      1, "admin", "123456", "localhost", Map("testConfig" -> "testValue"))
+
+    RestFrontendServiceSuite.withKyuubiRestServer {
+      (_, _, _, webTarget) =>
+        var response: Response = webTarget.path("api/v1/sessions")
+          .request(MediaType.APPLICATION_JSON_TYPE)
+          .post(Entity.entity(requestObj, MediaType.APPLICATION_JSON_TYPE))
+
+        val sessionHandle = response.readEntity(classOf[SessionHandle])
+        val serializedSessionHandle = s"${sessionHandle.identifier.publicId}|" +
+          s"${sessionHandle.identifier.secretId}|${sessionHandle.protocol.getValue}"
+
+        response = webTarget.path(s"api/v1/sessions/$serializedSessionHandle/info/13")
```

Comment:
We may need an API (in the following PR) to get all the enums of `TGetInfoType`, otherwise, no one knows what does the `13` mean and how many numbers like this (from the client). WDYT?

Reply:
OK

Reply:
sounds good

Reply:
FYI: added this API into the design doc.